app/main/views/users.py

```diff
@@ -450,17 +439,6 @@ def email_has_valid_buyer_domain_post():
     return jsonify(valid=domain_ok), 200
 
 
-# deprecated
-@main.route("/users/valid-admin-email", methods=["GET"])
-def email_is_valid_for_admin_user():
```

Comment:
This endpoint is used still in the [API client](https://github.com/alphagov/digitalmarketplace-apiclient/blob/master/dmapiclient/data.py#L541), which is used in two tests in admin FE (test_form_validators.py and test_admin_manager.py). I think we need to remove the tests first, then the client and finally this endpoint.